CI test randomly fails: test_compute_flow.py::test_compute_update_trusted_algorithm

Pre-requisite to this issue: [#938](https://github.com/oceanprotocol/ocean.py/issues/938) "Streamline Config: no config.ini file, no dict, just envvars." Then, doing this issue will be much, much simpler.

This test is now skipped, until the issue is fixed. Commit: https://github.com/oceanprotocol/ocean.py/commit/1c15c1d37617547aa33cb32bb17ca04b4138ac18

----

**Describe the bug**

I just created a branch, made a trivial change (added a space to a README), and created a [PR](https://github.com/oceanprotocol/ocean.py/pull/903) from it. 

This of course invoked the CI automated tests.

The automated test results [show two failures](https://github.com/oceanprotocol/ocean.py/runs/8213667665?check_suite_focus=true):
```text
=========================== short test summary info ============================
FAILED ocean_lib/data_provider/test/test_data_service_provider.py::test_get_root_uri
FAILED tests/integration/test_compute_flow.py::test_compute_update_trusted_algorithm
```

UPDATE Sep 19: the first test now passes. The second fail still exists, randomly.

--- a/ocean_lib/data_provider/test/test_data_service_provider.py
+++ b/ocean_lib/data_provider/test/test_data_service_provider.py
@@ -337,7 +337,7 @@ def test_provider_address_with_url():
 @pytest.mark.integration
 def test_get_root_uri():
     """Tests extraction of base URLs from various inputs."""
-    uri = "https://provider.mainnet.oceanprotocol.com"
+    uri = "https://v4.provider.mainnet.oceanprotocol.com"
     assert DataSP.is_valid_provider(uri)
     assert DataSP.get_root_uri(uri) == uri
     assert DataSP.get_root_uri("http://localhost:8030") == "http://localhost:8030"

--- a/setup.py
+++ b/setup.py
@@ -18,7 +18,7 @@ with open("README.md", encoding="utf8") as readme_file:
 # Installed by pip install ocean-lib
 # or pip install -e .
 install_requirements = [
-    "ocean-contracts==1.0.0",
+    "ocean-contracts==1.1.3",
     "coloredlogs",
     "pyopenssl",
     "PyJWT",  # not jwt

